lib/livebook/session.ex

```diff
@@ -1238,6 +1246,12 @@ defmodule Livebook.Session do
     {:noreply, handle_operation(state, operation)}
   end
 
+  def handle_cast({:select_hub, client_pid, id}, state) do
```

Comment:
Nit: similar operations are named `:set_*` so we can stick to that for consistency.